### PR TITLE
Add send_command_set and strip_commands methods

### PR DESCRIPTION
**Pull request by Boomerkuwanger**
This PR adds the following functionality:

- `send_command_set` function added that allows a list of commands to be sent and output collected after
- `strip_commands` method added that tries to find all commands in some output and remove them
- `_sanitize_output` method updated to call `strip_commands` when a command_set is passed in

### CODE REVIEW

File: netmiko/base_connection.py [hunk 1]
```diff
@@ -840,6 +843,50 @@ def send_command_expect(self, *args, **kwargs):
...
+            command_set = (command_set,)
+
+        if not hasattr(command_set, '__iter__'):
+            raise ValueError("Invalid argument passed into send_command_set")
+
+        if normalize:
+            command_set = [self.normalize_cmd(command) for command in command_set]
+
+        for cmd in command_set:
+            self.write_channel(self.normalize_cmd(cmd))
```
Comment (jjmanzer) [inline netmiko/base_connection.py]: This is a duplicate use of normalize_cmd().

File: netmiko/base_connection.py [hunk 2]
```diff
@@ -509,14 +509,17 @@ def _connect_params_dict(self):
...
         """Sanitize the output."""
         if self.ansi_escape_codes:
             output = self.strip_ansi_escape_codes(output)
         output = self.normalize_linefeeds(output)
         if strip_command and command_string:
             command_string = self.normalize_linefeeds(command_string)
             output = self.strip_command(command_string, output)
+        elif strip_command and command_set:
+            command_set = [self.normalize_linefeeds(command) for command in command_set]
+            output = self.strip_commands(command_string, output)
```
Comment (jjmanzer) [inline netmiko/base_connection.py]: Should be command_set.

**Comment by ktbyers** [comment]
I think we should return a list of results one for each command executed. 

This implies that I will change the behavior of this a bit.

**Comment by ktbyers** [comment]
We could do something like the following:

```
    def send_command_set(self, command_set, use_timing=False, **kwargs):
        """Generator to send commands down the SSH channel."""
        if isinstance(command_set, string_types):
            command_set = (command_set,)
        if not hasattr(command_set, '__iter__'):
            raise ValueError("Invalid argument passed into send_command_set")
        for cmd in command_set:
            if use_timing:
                yield self.send_command_timing(cmd, **kwargs)
            else:
                yield self.send_command(cmd, **kwargs)
```

And then using this would be as follows:

```
for output in net_connect.send_command_set(command_set):
    print(output)
```

Or:

```
output = list(net_connect.send_command_set(command_set))
```

**Comment by ktbyers** [comment]
I am going to close this...I am open to a slightly different implementation.

**Pull request closed without merging**